GithubHandler.py

```diff
@@ -144,11 +159,27 @@ def get_pr_deltas(self, pr):
         for f in file_changes:
             file_full_content = self.get_file_contents(f.filename, ref=pr.head.sha)
             full_file_delta = self.get_full_file_delta(f.filename, file_full_content)
-            file_delta = self.get_file_delta(f.patch, f.filename)
-            files_and_deltas += full_file_delta + "\n" + file_delta + "\n"
+            file_delta_str = self.get_file_delta(f.patch, f.filename)
+            files_and_deltas += full_file_delta + "\n" + file_delta_str + "\n"
 
         return files_and_deltas
 
+    def get_pr_deltas_in_list(self, pr):
+        file_changes = pr.get_files()
+        blocks = []
+        current_block = ""
+        for file in file_changes:
+            file_deltas = file.patch
+            for line in file_deltas.split("\n"):
+                if line.startswith("@@"):
+                    if current_block != "":
+                        blocks.append(current_block)
+                    current_block = line.split("@@")[-1] + "\n"
+
+                else:
+                    current_block += line + "\n"
+        return blocks
```

Comment:
The new `get_pr_deltas_in_list` method seems to be unused. If this method is not required, consider removing it to reduce dead code. If it's indeed used, make sure it is called correctly within the code.